.circleci/config.yml

```diff
@@ -90,6 +90,20 @@ commands:
           paths:
             - ".tox"
 
+  save_pip_cache:
+    description: "Save pip cache directory"
+    steps:
+      - save_cache:
+          key: pip-cache-{{ .Environment.CIRCLE_JOB }}-{{ .Environment.CIRCLE_NODE_INDEX }}-{{ checksum "riotfile.py" }}-{{ checksum "setup.py" }}
```

Comment:
is the node index consistent?

Reply:
We might need to ask CircleCI as the mechanism for how caches work across nodes is more complex than I had initially expected.

https://discuss.circleci.com/t/cache-save-restore-algorithm/759

Even if the node index is not consistent, each node's cache will add to its downloaded files. The problem would be that these caches get too large (500mb from CircleCI docs). In that case, we can update this cache key.